storybook/stories/Icon/index.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from "prop-types";
+import React from "react";
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    StyleSheet
+} from "react-native";
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+
+import design from "../../design-tokens";
+
+
+const Icon = props => {
+    const {
+        name,
+        size,
+        color,
+        underlayColor,
+        onPress
+    } = props;
+
+    return (
+        <View>
+          <TouchableOpacity
+            underlayColor = {design.color[underlayColor]}
```

Comment:
Does TouchableOpacity use underlay color?

Reply:
No it does not; this is left over from when I was using TouchableHighlight (didn't like the way it looked).  I'll cut it out for the next push